R/wwinference.R

```diff
@@ -62,10 +62,7 @@
 #'
 wwinference <- function(ww_data,
                         count_data,
-                        model_spec = wwinference::get_model_spec(
-                          forecast_date =
-                            "2023-12-06"
-                        ),
+                        model_spec = wwinference::get_model_spec(),
```

Comment:
```suggestion
                        model_spec,
```

Reply:
Am going to keep this with default values for now!